MediaProcessor/tests/TestUtils.cpp

```diff
@@ -79,12 +79,12 @@ bool CompareFiles::compareFilesByteByByte(const fs::path& filePath1, const fs::p
 bool CompareFiles::compareAudioFiles(const fs::path& filePath1, const fs::path& filePath2,
                                      double tolerance, size_t chunkSize) {
     SF_INFO sfInfo1, sfInfo2;
-    SNDFILE* sndFile1 = sf_open(filePath1.c_str(), SFM_READ, &sfInfo1);
+    SNDFILE* sndFile1 = sf_open(filePath1.string().c_str(), SFM_READ, &sfInfo1);
```

Comment:
In that case, adding a brief comment here would be good as this can easily be forgotten and removed during a refactor. A bit surprising though, as I had no issue running this natively on Windows 10.